### PR TITLE
{chem}[fosscuda/2020b] cryoDRGN v0.3.2 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/cryoDRGN/cryoDRGN-0.3.2-fosscuda-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+exts_list = [
+    ('colorlover', '0.3.0', {
+        'checksums': ['b8fb7246ab46e1f5e6715649453c1762e245a515de5ff2d2b4aab7a6e67fa4e2'],
+    }),
+    ('retrying', '1.3.3', {
+        'checksums': ['08c039560a6da2fe4f2c426d0766e284d3b736e355f8dd24b37367b0bb41973b'],
+    }),
+    ('plotly', '4.14.3', {
+        'checksums': ['7d8aaeed392e82fb8e0e48899f2d3d957b12327f9d38cdd5802bc574a8a39d91'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/c/cryoDRGN/cryoDRGN-0.3.2-fosscuda-2020b.eb:39]: Replace this with a dependency on https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/plotly.py/plotly.py-4.14.3-GCCcore-10.2.0.eb

File: easybuild/easyconfigs/c/cryoDRGN/cryoDRGN-0.3.2-fosscuda-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    ('pynndescent', '0.5.2', {
+        'checksums': ['d9fd22210b8d64368376ff392e876fb72fe3cda282396cfa6a59440ab6600771'],
+    }),
+    ('umap-learn', '0.5.1', {
+        'checksums': ['3e3e5e526109866012a9da79f423c922edc379c6cac9bf65ea08fbb9dd93ff3a'],
+        'modulename': 'umap',
+    }),
+    ('seaborn', '0.11.1', {
+        'checksums': ['44e78eaed937c5a87fc7a892c329a7cc091060b67ebd1d0d306b446a74ba01ad'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/c/cryoDRGN/cryoDRGN-0.3.2-fosscuda-2020b.eb:52]: Replace this with a dependency on https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/s/Seaborn/Seaborn-0.11.1-fosscuda-2020b.eb

File: easybuild/easyconfigs/c/cryoDRGN/cryoDRGN-0.3.2-fosscuda-2020b.eb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/zhonge/cryodrgn/archive'],
+        'checksums': ['1efecc2a0add2392411076082c122039a490f5ddf3e01f1da38aea1a81592077'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```
Comment (branfosj) [inline easybuild/easyconfigs/c/cryoDRGN/cryoDRGN-0.3.2-fosscuda-2020b.eb:63]: We can enhance the sanity check by replacing the `sanity_check_paths` with
```
sanity_check_commands = ["%(namelower)s --help"]
```

The default PythonBundle sanity check will test for the existence of the directory and the sanity check command will test the existence and function of `cryodrgn`.

**Review by branfosj — CHANGES_REQUESTED** [review]
I've made some minor suggestions for improvement.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3304.joltik.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/3ff7b8e6889e7dbc6d7eff7a77d32f88 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**